app/messages.py

```diff
@@ -1,3 +1,4 @@
+import app.api.validations.user as user_fields_validation_constants
```

Comment:
could you import directly the constants, it would make your line 14 much shorter and also making sure you only import what you need

Reply:
@isabelcosta - yes sure, thanks for the review :)